packages/components/inputs/multiline-text-input/src/multiline-text-input.tsx

```diff
@@ -99,6 +108,27 @@ export type TMultilineTextInputProps = {
     | 16
     | 'scale'
     | 'auto';
+
+  /**
+   * Custom action icon to be displayed on the right side of the input.
+   */
+  rightActionIcon?: ReactElement;
+  /**
+   * Props for the right-action icon-button. Required when rightActionIcon is provided.
+   * At least a `label` and an `onClick` prop/function need to be provided.
+   */
+  rightActionProps?: TSecondaryButtonIconProps;
+  /**
+   * Set this to `true` to reduce the paddings of the input allowing the input to display
+   * more data in less space.
+   *
+   */
+  isCondensed?: boolean;
```

Comment:
I don't know if this is a thing, but if the `isCondensed` property changes from`false` to `true` during runtime , it does not seem to re-trigger some height-calculations and the input is ending up in this state:

![image](https://github.com/commercetools/ui-kit/assets/169909964/7a5b7906-3be3-4a72-8c42-e58359e283a2)

Reply:
Thank you @misama-ct 
Exactly! a question I would also like to ask, especially from your suggestion with the isCondensed height change. Since this is a multiline text input, should the condensed value just work for when there is 1 row of text, what should happen when we have multiple rows? 🤔

My initial thought was basically to just reduce the text size and the padding but I guess we would need @FilPob 's help here 🙏🏾

Reply:
> should the condensed value just work for when there is 1 row of text, what should happen when we have multiple rows?

I don't think so. And if it starts in the `isCondensed` state everything looks okay (try setting it to true in the sb story and reload). 

So basically this only happens if `isCondensed` changes to true *after* the component has already been rendered once in non-condensed state. The `TextareaAutosize` sets the fixed height, so this component has to be made aware of the size-change somehow.

The only solution I have right now is adding a key-prop to the`TextareaAutosize` usage [here](https://github.com/commercetools/ui-kit/blob/56a2cfeb3a44315a51578e15a17facdc51035569/packages/components/inputs/input-utils/src/multiline-input/multiline-input.tsx#L81), essentially forcing a complete re-render of the `TextareaAutosize` component whenever the prop changes.

```
<TextareaAutosize ...  key={props.isCondensed}/>
```

Reply:
I see your point. Thanks for the pointer 🙏🏾 

What I also noticed is that the `TextareaAutosize` uses a  `cacheMeasurements` prop to compare before rendering. I think. a sort of optimisation feature. If this value does not "change", then it uses the initial value without any further computing. So I am not so sure how deep it goes with this comparisons but it currently seems like it does not notice the prop change. 
setting `cacheMeasurements={false}` from the `MultilineTextInput` gives us the behaviour we want. 
The question is what significance is what impact is it, turning off `cacheMeasurements` ?

Reply:
From what I am reading, this feature would be useful if we make freequent updates that change the measurements of properties like scroll and text area height rapidly like in a live chat or real-time collaborations. 
I don't think it would hurt to turn it off, in my opinion.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2845/commits/fce428fecc73f6056c9da3533c93a920ad20c49e

@commercetools/shield-team-design-system what do you think of this change please?

Reply:
I agree. By default caching is not turned on either.